R/execution.r

```diff
@@ -133,6 +134,14 @@ execute = function(request) {
     assign('quit', quit, envir = .GlobalEnv)
     assign('q',    quit, envir = .GlobalEnv)
     
+    send_error_msg <- function(msg){
```

Comment:
can you put this as method into the executor (and add a space here)?

maybe like `send_error_msg(request, msg)`? `if (!silent)` would have o go outside of this then though, …

i prefer not to create too many functions in procedural pieces of code.